programmers/서울에서-김서방-찾기.js

```diff
@@ -0,0 +1,9 @@
+function solution(seoul) {
+  var answer = "";
```

Comment:
var는 es6에서 let과 const가 나온 이후 잘 사용되지 않습니다.
변수 선언은 let과 const로만 하는게 좋은데 
상수라면 const
변수라면 let으로 선언하면 됩니다.

[var let const](https://velog.io/@bathingape/JavaScript-var-let-const-%EC%B0%A8%EC%9D%B4%EC%A0%90)
```suggestion
  let answer = "";
```